apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -153,8 +161,33 @@ const AllResidentBillingReceiptsService = new GQLCustomSchema('AllResidentBillin
                     deletedAt: null,
                 })
 
+                // cache billing receipt for calculation of is payable field
+                const minPeriod = min(receiptsForConsumer.map(item => item.period))
+                const nextPeriodBillingReceiptsCache = await find('BillingReceipt', {
+                    account: { id_in: [...new Set(receiptsForConsumer.map(item => item.account.id))], deletedAt: null },
```

Comment:
BillingAccount is a deprecated field. You need to use accountNumber and organizationId